src/schema/v2/__tests__/filterArtworksConnection.test.ts

```diff
@@ -0,0 +1,533 @@
+/* eslint-disable promise/always-return */
+import { runQuery } from "schema/v2/test/utils"
+import { toGlobalId } from "graphql-relay"
+import gql from "lib/gql"
+import sinon from "sinon"
+
+describe("artworksConnection", () => {
```

Comment:
This ports to [the old JS spec](https://github.com/artsy/metaphysics/pull/2130/files#diff-58cdd201ce1c1409ecd5a2a9d001aad6) to TS and also:
- adds async/await where applicable
- uses the `gql` helper
- uses `jest.fn()` for newer mocks